### PR TITLE
feat(timepicker): add aria-label to input

### LINKED ISSUE #4149: feat(timepicker): add aria-label to inputs
Need add aria-attributes to HTML code in `timepicker`component.

- description of `aria-label`: https://www.w3.org/TR/wai-aria-1.1/#aria-label

Took as a basis - https://ng-bootstrap.github.io/#/components/timepicker/examples

### PR DESCRIPTION
**Pull request by tarusin**
Closes #4149

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated API documentation.

### CODE REVIEW

File: src/timepicker/timepicker.component.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
              (wheel)="prevDef($event);changeHours(hourStep * wheelSign($event), 'wheel')"
              (keydown.ArrowUp)="changeHours(hourStep, 'key')"
              (keydown.ArrowDown)="changeHours(-hourStep, 'key')"
-             (change)="updateHours($event.target.value)"></td>
+             (change)="updateHours($event.target.value)" aria-label="hours"></td>
```
Comment (IlyaSurmay) [inline src/timepicker/timepicker.component.html]: make inputs from these labels

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: src/timepicker/timepicker.component.ts [hunk 2]
```diff
@@ -115,6 +115,11 @@ export class TimepickerComponent
   seconds: string;
   meridian: string;
 
+  // aria-label variables
+  labelHours = 'hours';
```
Comment (IlyaSurmay) [inline src/timepicker/timepicker.component.ts]: `@Input` ?

Also add these new options to the timepicker config

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

File: src/timepicker/timepicker.component.ts [hunk 3]
```diff
@@ -105,6 +105,12 @@ export class TimepickerComponent
   @Input() min: Date;
   /** maximum time user can select */
   @Input() max: Date;
+  /** hours aria label  */
+  @Input() labelHours: string;
```
Comment (valorkin) [inline src/timepicker/timepicker.component.ts]: it should not be an input,
please remove `@Input()` decorator

**Review by valorkin — CHANGES_REQUESTED** [review]

File: src/timepicker/timepicker.component.ts (hunk 2 above)
Comment (valorkin) [reply src/timepicker/timepicker.component.ts]: yes, should be configurable via config, and config should be applied before ngOnInit

**Review by valorkin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**